blacksauce/src/styles/grid/_404PgGrid.scss

```diff
@@ -0,0 +1,30 @@
+.grid--404Pg {
+    background-color: black;
```

Comment:
Many of the grid styles in this directory are just that -- about the layout and placement of items. I think that's a good division of stylesheets and would recommend taking styles about colors and text alignment (see later in this stylesheet) to another place.